### ISSUE TITLE
[Bug Report][4.1.2] Accessibility: VDateInput input element have role but not required aria attributes (following #22017)

### ISSUE THREAD

**Issue by Phoenix751** [vuetifyjs/vuetify#22977]
### Environment
**Vuetify Version:** 4.1.2
**Vue Version:** 3.5.32
**OS:** Linux

### Steps to reproduce
1. Add VDateInput

### Expected Behavior
The input element inside the <v-select> should have role="combobox" and aria-expanded & aria-controls attributes.

### Actual Behavior
The input element have the role but not the required combobox attributes.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqlVU1vEzEQ/StmOawjJd5CQYiohSIEAokDAgSHpoftejYx+GOxvaFRlf/O+GO327RClThYWc88z3szHk/OrwuhOmP9QtUd++mMLpbF9UoTssoOtyqWJFqCbdtD2K+KjfedW1ZVwzUe4yDF1jINvtKdqs4QVtlee6FgwY06O2bP2fHLigvnp3YGTi0urfnjwGKUVTGf8FRo3IJdWNAcLNiH8h4cu8V94Psnvxft7gGcGXn2jD1hTxNLNi1kfekCRQwdIu9Xel/MCyn0L3dQ7MbFQp8PAv6LMwYLkS4SobNN9abrGIKQ8cSD6mTt4VWAnGwXddfFz7hpjPa10GCzKRo5ohdCd70n1QCtDrHBkkKdVBMK3LrGis4TB75PTKm1yDWx0JI9aa1RpER1ZYATgnGdJ8qtyWlA0PIDSGnID2Mlf1TOAkEKicFyegp1MO8wvTF2YwEVYN4HDCMg12x0s6GKN6BwenDmCt440RZTiqqTZswfNY/MFBfKRStDfyvWDKw19kOtuQSLSCCnr2K6RkLyUcAD6UjvgGZFo00ZfD60fIzfoRA5+ygjpW9BmS189TsJ3+o1Rcxn1C8csFpKei5Nzd/jtdHZnITvj4Gczi4iQdvrxgujsejTIITO0gRIOXq0nBJuml6B9ux3D3b3FSQ0HtWXj4dO9BvAV+5CDLcB8EEu3nxL6COMMEMO39t43bhlXY2P0r+TeAZjJv63GyE5DWBs41v6gvKoLkYmdINdkjWmsETDH5ITp9QC1ncLc3T+RJmzUPQ80lJK4UVOc0oXmNXQMriTfBKhqE8ivJwkN3EGMegNPxNrzfm7LUb7hK8V8MlgVEyiDJqiuGn4u9jYGhEcE0jYUe6l4TuGHYFjjYYA0b+/t2rh7sf7zLW6XU1aDrOnRaxja2PWEupO4FwxqsLZ8rqtlZC70y/m0nizfHJ0ND/G9QzXc1wvcL08OgoFu8sf++2hAu4f8oqLKO3sBbsKcvDte7CilhycWGsRGJjCcYC+LCI/k9ya986J7wfD4GYUrDRcRWjqlWFqDM88H6Qxodj04/8lh7bupf+WjNhG640v839MVcVLwl24qGI/L9IfFUqNY72Yt7V0cHt4D7asoZhfj5/LfGx/8RfJo7Lv)

### Other Comments

Already reported in #22522. It es recognized as Textbox but is has `aria-expand` flagged as not allowed because it's missing `role="combobox"`
